.codeclimate.yml

```diff
@@ -20,6 +20,11 @@ plugins:
         enabled: false
       DeclarationOrder:
         enabled: false
+      SelectorFormat:
+        disabled: true
+      NestingDepth:
+        enabled: false
```

Comment:
I wanted to apply [BEM](http://getbem.com/introduction/) methodology to define CSS rules and nesting and class format was not allowing me. I think using something like BEM improve CSS qualify because it define clearly rules specificity. 

In a ideal word we should use Tailwind or a React components with CSS-in-JS which makes CSS more manageable but having global css BEM is a good start